accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -391,6 +391,13 @@ continuation.
 
 #### Late fields and variables
 
+A non-local variable declaration of the form `late final v;`, `late final T v;`,
+`late final v = e;`, or `late final T v = e;`, or the same forms with `static`
+prepended, implicitly induces a getter into the enclosing scope. Moreover, the
+forms that do not include an initializing expression `e` induce a setter as
+well. For other variable declarations including `late`, the same getters and
```

Comment:
Maybe:

   other variable declarations &mapsto; non-`final` non-local variable declarations 

It's unclear what "other" covers. If my suggestion is not correct, then it definitely needs to be spelled out.